### ISSUE TITLE
Scan using Android Rear Camera result in inverted camera

### ISSUE THREAD

**Issue by desmond0412** [schmich/instascan#70]
Hi, i am trying your live demo using android rear camera. It worked using inverted QR code, is there any way i can call the function to invert it back to normal?

Thanks

**Comment by Tieantono** [comment]
Yes please, I also want to know the solution too. The device I'm using is Samsung J7 Pro, and I'm using Firefox for the browser.

**Comment by alecpo** [comment]
Yes please, I also want to know how to fix it! Help us Schmich ! I'm using LG G3 and opera/google brower.

**Comment by tsklee** [comment]
Yes plz help with the inverted iaaue

**Comment by michaelneu** [comment]
"Temporary fix": There's a `mirror` option, which you can pass to the `Scanner`'s constructor like this:

```js
let scanner = new Instascan.Scanner({
  video: document.getElementById('preview'),
  mirror: false // prevents the video to be mirrored
});
```

Using this option, the rear facing camera will provide the correct, non-mirrored image.
